payments-core/build.gradle

```diff
@@ -39,6 +39,7 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$kotlinCoroutinesVersion"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$kotlinCoroutinesVersion"
+    implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
```

Comment:
Why do we need to add this? According to the [docs](https://developer.android.com/jetpack/androidx/releases/lifecycle#declaring_dependencies), this dependency is deprecated.

Reply:
Android Studio added it when I was figuring out how to create the ViewModel in AddCardFragment, but you're right, the way I ended up doing it doesn't need this.